bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperClientStats.java

```diff
@@ -66,9 +66,14 @@
     String TIMEOUT_GET_BOOKIE_INFO = "TIMEOUT_GET_BOOKIE_INFO";
     String CHANNEL_START_TLS_OP = "START_TLS";
     String CHANNEL_TIMEOUT_START_TLS_OP = "TIMEOUT_START_TLS";
+
     String NETTY_EXCEPTION_CNT = "NETTY_EXCEPTION_CNT";
     String CLIENT_CONNECT_TIMER = "CLIENT_CONNECT_TIMER";
     String ADD_OP_OUTSTANDING = "ADD_OP_OUTSTANDING";
     String READ_OP_OUTSTANDING = "READ_OP_OUTSTANDING";
     String NETTY_OPS = "NETTY_OPS";
+    String ACTIVE_NON_TLS_CHANNEL_COUNTER = "ACTIVE_NON_TLS_CHANNEL_COUNTER";
+    String ACTIVE_TLS_CHANNEL_COUNTER = "ACTIVE_TLS_CHANNEL_COUNTER";
+    String FAILED_CONNECTION_COUNTER = "FAILED_CONNECTION_COUNTER";
+    String FAILED_TLS_HANDSHAKE_COUNTER = "FAILED_TLS_HANDSHAKE_COUNTER";
```

Comment:
I don't see a stat on how many times we connected to a bookie. I guess we can infer it through failed-connection-counter but on a flaky network, IMO it is good to have this counter too.

Reply:
discussed this with @jvrao further. The connectTimer in PCBC will track the connection attempts.

Reply:
yeah; that's good.